ui/src/shared/constants/index.js

```diff
@@ -468,6 +468,8 @@ export const STROKE_WIDTH = {
   light: 1.5,
 }
 
+export const HEARTBEAT_INTERVAL = 10000 // ms
```

Comment:
Ack `document.cookie` does eat all that info doesn't it 😞 . I think we could get this off the response from the heartbeat request itself but after dropping in a few breakpoints on this branch, it looks like `axios` abstracts this away from us.

This is probably fine for now... I don't want to spend a ridiculous amount of time on this unless it becomes an actual problem for someone.